mvnw.cmd

```diff
@@ -58,6 +58,7 @@ set ERROR_CODE=0
 @setlocal
 
 @REM ==== START VALIDATION ====
+set JAVA_HOME=C:\Program Files\Java\jdk1.8.0_301
```

Comment:
Why was this change needed?

Reply:
Oh, it reflects my intention to use it personally